test/FeeVault.t.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {ChannelOpenTestBaseSetup} from "./Dispatcher/Dispatcher.t.sol";
+import {Ownable} from "@openzeppelin/contracts/access/Ownable.sol";
+import "forge-std/Test.sol";
+
+contract FeeVaultTest is ChannelOpenTestBaseSetup {
+    address notOwner = vm.addr(2);
+    address owner;
+    Ownable feeVaultOwnable; // FeeVault address but from Ownable interface
+    uint256 feePerGreet = 600_000 * 60 gwei + 700_000 * 70 gwei;
+
+    function setUp() public override {
+        super.setUp();
+
+        feeVaultOwnable = Ownable(address(feeVault));
+        greetMarsWithFee();
+    }
+
+    // Fuzz test for always reverting if incorrect fee is sent.
+    function testFuzz_sendPacket_FeesAddUpToValue(
+        uint104 gasFee1,
+        uint104 gasFee2,
+        uint104 gasLimit1,
+        uint104 gasLimit2,
+        bool shouldRevert,
+        uint56 fuzz
+    ) public {
+        gasFee1 = uint104(bound(gasFee1, 1, 2 ** 104 - 1));
+        gasLimit1 = uint104(bound(gasFee1, 1, 2 ** 104 - 1));
```

Comment:
`gasLimit1 = gasFee1` ?

Reply:
now that I'm revisiting this, we can actually get rid of these bounds altogether

Reply:
yep, thanks!